package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -7,6 +7,32 @@ DATA_DISK_FSLABEL="HARV_LH_DEFAULT"
 
 clear_disk_label()
 {
+    # For internal automation environments we need to be able to wipe all disks on the node
+    # to ensure that host does not boot off partitions from another disk in the host
+    # lsblk produces the following json output, which we filter using jq to identify disks
+    # which we will be wiping
+    # lsblk -d -n -J -o NAME,TYPE
+    #{
+    #"blockdevices": [
+    #  {
+    #     "name": "loop0",
+    #     "type": "loop"
+    #  },{
+    #     "name": "sr0",
+    #     "type": "rom"
+    #  },{
+    #     "name": "vda",
+    #     "type": "disk"
+    #     }
+    #   ]
+    # }
+    if [ "$HARVESTER_WIPE_DISKS" == "true" ]; then
+       echo "wiping all disks"
+       for disk in $(lsblk -d -n -J -o NAME,TYPE | jq -r '.blockdevices[] | select(.type == "disk") | .name')
+       do
+         sgdisk -Z /dev/$disk
+       done
```

Comment:
just a nit: we can do `partprobe` here. For rescanning the whole partition.

Reply:
done